app/src/components/form/LModal.vue

```diff
@@ -0,0 +1,64 @@
+<script setup lang="ts">
+import LButton from "../button/LButton.vue";
+
+type Props = {
+    isVisible: boolean;
+    heading: string;
+    bLeftText?: string;
+    bRightText?: string;
+};
+defineProps<Props>();
+
+const emit = defineEmits(["close", "b-left-click", "b-right-click"]);
```

Comment:
If we remove all buttons from the footer are (see next comment) we only need the "close" event (which the developer can choose to implement or not)